e2e/tests/expression/validationExpression.spec.ts

```diff
@@ -0,0 +1,100 @@
+import { test, expect } from "@playwright/test";
+
+test("test", async ({ page }) => {
```

Comment:
i think we need to name each test something descriptive , when it fail the name is used i the output ( as you can have multiple tests per file/spec)